translations/harbour-seaprint-de.ts

```diff
@@ -4606,379 +4606,379 @@ auf diesem Drucker</translation>
     </message>
     <message>
         <source>Stitcher Interlock Open</source>
-        <translation type="unfinished"></translation>
+        <translation>Verriegelung des Hefters offen</translation>
```

Comment:
Hefter sounds like Clamper/Stapler to me... but it is also Stitcher?

Reply:
yes it is the same